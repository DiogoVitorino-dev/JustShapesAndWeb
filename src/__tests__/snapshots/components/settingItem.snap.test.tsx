import { render } from "@testing-library/react-native";

import SettingItem from "@/components/settings/settingItem";

describe("Setting Item - Snapshot component test", () => {
  it("should render correctly", () => {
    const tree = render(
      <SettingItem title="test" icon="add" to="./" />,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
