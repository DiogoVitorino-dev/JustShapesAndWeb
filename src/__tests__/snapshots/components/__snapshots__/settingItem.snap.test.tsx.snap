// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Setting Item - Snapshot component test should render correctly 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={true}
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
>
  <View
    style={
      [
        {
          "backgroundColor": "rgb(3, 38, 25)",
        },
        {
          "alignItems": "center",
          "flexDirection": "row",
          "paddingBottom": 16,
        },
      ]
    }
  >
    <Text />
    <Text
      style={
        [
          {
            "color": "rgb(245, 245, 245)",
            "fontSize": 22,
          },
          [
            {
              "height": "auto",
            },
            {
              "fontFamily": "MajorMonoDisplay",
            },
          ],
        ]
      }
    >
      test
    </Text>
  </View>
</View>
`;
